pkgs/servers/http/router/librusty_v8.nix

```diff
@@ -0,0 +1,21 @@
+# auto-generated file -- DO NOT EDIT!
```

Comment:
How is that auto generated?

Reply:
No idea. That was in pkgs/development/web/deno/librusty_v8.nix. I'll remove the comment.

Reply:
I think it is generated from their update script